### PR TITLE
mprven may be tied high or low.

### PR DESCRIPTION
**Pull request by timsifive**

### CODE REVIEW

File: xml/core_registers.xml [hunk 1]
```diff
@@ -87,11 +87,12 @@
...
-            When 1, \Fmprv in \Rmstatus takes effect during debug mode.
-            When 0, it is ignored during debug mode.
-            Implementing this bit is optional.
-            If not implemented it should be tied to 0.
+        <field name="mprven" bits="4" access="R/W" reset="Preset">
+            0: \Fmprv in \Rmstatus is ignored in Debug Mode.
+
+            1: \Fmprv in \Rmstatus takes effect in Debug Mode.
+
+            Implementing this bit is optional. It may be tied to either 0 or 1.
```
Comment (mwachs5) [inline xml/core_registers.xml]: I'm fine with this, but is there a different  `access=R/W` that basically says this?

Comment (timsifive) [reply xml/core_registers.xml]: Nothing that we're currently using. "R/W?" might mean readable and optionally writable. I just made that up. I'm not sure how intuitive it is.

**Comment by allenjbaum** [comment]
Can’t you just define it as WARL with preset initial value?

-Allen

**Comment by timsifive** [comment]
Yes. Thanks for pointing that out.

**Pull request merged**